src/commands/refocus-collector-start.js

```diff
@@ -19,7 +19,7 @@ program
   .option('-n, --collectorName <collectorName>', 'The name of the collector to be started')
   .option('-n, --refocusUrl <refocusUrl>', 'The url of the refocus instance this collector' +
     ' will send to')
-.option('-n, --accessToken <accessToken>', 'A valid refocus token')
+  .option('-n, --accessToken <accessToken>', 'A valid refocus token')
```

Comment:
I think you can only have one `-n` option

Reply:
I did not change this. Just fixed style. @iamigo

Reply:
Yeah, but can you please update them while you're modifying this file?
use -n for collectorName
use -u for refocusUrl
use -t for accessToken

Reply:
Done 👍